clang/test/CodeGenCXX/typeinfo-with-address-space.cpp

```diff
@@ -14,8 +14,8 @@ class B : A {
 // NO-AS: @_ZTISt9type_info = external constant ptr
 // AS: @_ZTIi = external addrspace(1) constant ptr addrspace(1)
 // NO-AS: @_ZTIi = external constant ptr
-// AS: @_ZTVN10__cxxabiv117__class_type_infoE = external addrspace(1) global ptr addrspace(1)
-// NO-AS: @_ZTVN10__cxxabiv117__class_type_infoE = external global ptr
+// AS: @_ZTVN10__cxxabiv117__class_type_infoE = external addrspace(1) global [0 x ptr]
```

Comment:
Maybe this should be `[0 x ptr addrspace(1)]`?  Not sure that makes any semantic difference, but it would be more clear.  (So use `llvm::ArrayType::get(CGM.GlobalsInt8PtrTy, 0)`.)

Reply:
ok, changed!